### PR TITLE
Update production-offline-deployment-using-tiup.md

### PR DESCRIPTION
**Pull request by ireneontheway**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: [pingcap/docs-cn#4015](https://github.com/pingcap/docs-cn/pull/4015)
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: production-offline-deployment-using-tiup.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ This document describes how to deploy a TiDB cluster offline using TiUP.
 
 ### Option 1: Download the official TiUP offline component package
 
-Download the prepared TiUP offline package from the [Download TiDB](https://pingcap.com/download) page.
+Download the offline mirror package of TiDB server (including the TiUP offline component package) from the [Download TiDB](https://pingcap.com/download-cn/) page.
```
Comment (yikeke) [inline production-offline-deployment-using-tiup.md:15]:
```suggestion
Download the offline mirror package of the TiDB server (including the TiUP offline component package) from the [Download TiDB](https://pingcap.com/download/) page.
```
For readers who speak English, give them the English version of the link. Be cautious of this kind of translation and use your critical thinking to get close to the truth! 😉 @ireneontheway

**Comment by ireneontheway** [comment]
@yikeke PTAL

**Pull request merged**